js/components/Todo/TodoList.js

```diff
@@ -0,0 +1,150 @@
+import Component from "../../core/component.js";
+import store from '../../store/index.js';
+import {
+    deleteTeamToMemberToTodoItem,
+    putTeamToMemberToTodoItemToContents,
+    putTeamToMemberToTodoItemToPriority,
+    putTeamToMemberToTodoItemToToggle
+} from "../../service/TodoApi.js";
+import {filter,priority, eventType, keyboardKey} from "../../constants/constants.js";
+
+export default class TodoList extends Component {
+    todoListTemplate = (filterType, memberId, {todoList}) => {
+        if (todoList && todoList.length > 0) {
```

Comment:
```suggestion
        if (todoList && todoList.length <= 0) return '할 일을 입력해주세요'
```
if/switch가 감싸는 부분은 가능하면 최대한 적게 유지해주세요!